modules/profile/manifests/ros/base.pp

```diff
@@ -42,7 +42,7 @@
     $defaults = {
       'ensure' => 'present',
     }
-    create_resources(ssh_authorized_key, hiera('ssh_keys'), $defaults)
+    # create_resources(ssh_authorized_key, hiera('ssh_keys'), $defaults)
```

Comment:
This needs to be reverted, only here to be able to launch the code without keys.